commands/operator-sdk/cmd/up/local.go

```diff
@@ -28,13 +28,15 @@ kubernetes cluster using a kubeconfig file.
 
 	upLocalCmd.Flags().StringVar(&kubeConfig, "kubeconfig", "", "The file path to kubernetes configuration file; defaults to $HOME/.kube/config")
 	upLocalCmd.Flags().StringVar(&operatorFlags, "operator-flags", "", "The flags that the operator needs. Example: \"--flag1 value1 --flag2=value2\"")
+	upLocalCmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "The namespace where the operator will watch.")
```

Comment:
I am debating whether should we add a short named flag for namespace because none of the other flags have a short version.

I'd suggest we don't add it now.  Maybe we can revisit adding the short flags in a future pr.

Suggest to avoid using the word `will` and keep description in present tense: `The namespace where the operator will watch.` -> `The namespace where the operator watches for changes`?